sample/Sample.Microsoft.Data.SqlClient/Program.cs

```diff
@@ -0,0 +1,62 @@
+using Elastic.Apm;
+using Elastic.Apm.Logging;
+using Elastic.Apm.SqlClient;
+using Elastic.Apm.Tests.Utilities;
+using Microsoft.Data.SqlClient;
+using Testcontainers.MsSql;
+
+var agentConfig = new MockConfiguration(centralConfig: "false");
+Agent.Setup(new AgentComponents(ConsoleLogger.LoggerOrDefault(LogLevel.Trace), agentConfig));
+Agent.Subscribe(new SqlClientDiagnosticSubscriber());
+using var instance = Agent.Instance;
+
+Console.WriteLine("Starting mssql container");
+await using var msSql = new MsSqlBuilder().Build();
+await msSql.StartAsync();
+var connectionString = msSql.GetConnectionString();
+
+Console.WriteLine("Performing database calls");
+
+var transaction = Agent.Tracer.StartTransaction("test-sql-client-calls", "app");
+DoSqlClientCall(connectionString);
+transaction.End();
+
+Console.WriteLine("Done!");
+
+static void DoSqlClientCall(string connectionString)
```

Comment:
Could we move this into a test? From what I understand, we do not run this currently anywhere. I think `Elastic.Apm.SqlClient.Tests` would be a better place for this.